src/components/themed/ExchangedFlipInput2.tsx

```diff
@@ -223,6 +216,8 @@ export const ExchangedFlipInput2 = React.memo((props: ExchangedFlipInputProps) =
   )
 })
 
+export const ExchangedFlipInput2 = React.memo(ExchangedFlipInput2Component)
```

Comment:
We typically put this line at the very bottom of the file, as a hold-over from the old connector days. I'm not sure if this makes sense anymore, so I'll leave it up to you.